id-repository-default.properties

```diff
@@ -427,4 +427,13 @@ mosip.role.idrepo.vid.postvidreactivate=RESIDENT,ID_REPOSITORY
 mosip.role.idrepo.vid.postdraftvid=REGISTRATION_PROCESSOR,ID_REPOSITORY
 mosip.mask.function.identityAttributes=convertToMaskDataFormat
 
-mosip.role.idrepo.credentialrequest.postv2requestgeneratorrid=CREDENTIAL_REQUEST,ID_REPOSITORY
\ No newline at end of file
+mosip.role.idrepo.credentialrequest.postv2requestgeneratorrid=CREDENTIAL_REQUEST,ID_REPOSITORY
+
+mosip.idrepo.credential.request.enable-convention-based-id=true
+mosip.idrepo.credential-request-v2.rest.uri=${mosip.idrepo.credrequest.generator.url}/v1/credentialrequest/v2/requestgenerator/{rid}
+mosip.idrepo.credential-request-v2.rest.httpMethod=POST
+mosip.idrepo.credential-request-v2.rest.headers.mediaType=application/json
+mosip.idrepo.credential-request-v2.rest.timeout=100
+
+mosip.idrepo.identity.disable-uin-based-credential-request=false
+mosip.identity.fieldid.handle-postfix.mapping={'phone':'@phone'}
```

Comment:
let us add a comment on this property as below
"Field Id as in the identity schema will be the key and value is the actual postfix to append. Empty values are also supported. This configuration is considered only when mentioned fieldId is marked to be a handle in the  in the identity schema and it is one of the selectedHandle in the ID-object."